### PR TITLE
add ipynb for vid2desc

### PR DESCRIPTION
**Pull request by deep-diver**
ipynb notebook to demonstrate how to ask about video clip to Gemini-1.0-Pro-Vision model on Vertex AI.

### CODE REVIEW

File: notebooks/Vid2Desc_Gemini_1_0_Pro_Vision.ipynb [hunk 1]
```diff
@@ -0,0 +1,378 @@
+{
```
Comment (sayakpaul) [inline notebooks/Vid2Desc_Gemini_1_0_Pro_Vision.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #12.            raise ValueError("Either gcs or base64_encoded must be provided")
</pre><p> </p><p><code></code><code>python</p><p>raise ValueError("Either a GCS bucket path or base64_encoded string of the video must be provided")</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-diver/Vid2Persona/pull/1/discussion/#comment-1512307603'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (sayakpaul) [inline notebooks/Vid2Desc_Gemini_1_0_Pro_Vision.ipynb:1]: <p>It's a single-shot caption summarizing the video, yeah?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-diver/Vid2Persona/pull/1/discussion/#comment-1512307606'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

**Review by sayakpaul — APPROVED** [review]
Very cool.

**Comment by sayakpaul** [comment]
Once this is done, I would like to give the following a try: 

> based on the description, ask Large Language Model to pretend to be the person

> then, chatting with that personality

Could you provide me a with sample descriptions that have person presence inside of them?

File: notebooks/Vid2Desc_Gemini_1_0_Pro_Vision.ipynb (hunk 1 above)
Comment (deep-diver) [reply notebooks/Vid2Desc_Gemini_1_0_Pro_Vision.ipynb:1]: yes it is!

**Comment by deep-diver** [comment]
@sayakpaul sure sayak!
let me create the other PR to upload sample video and sample description for that!

**Pull request merged**